pkg/rebuild/debian/strategy.go

```diff
@@ -137,6 +148,13 @@ var toolkit = []*flow.Tool{
 			Needs: []string{},
 		}},
 	},
+	{
+		Name: "debian/patch/debrebuild",
+		Steps: []flow.Step{{
+			Runs:  "echo 'QEAgLTcyMyw2ICs3MjMsNyBAQAogICAgICAgICAgICAgICAuIChTdHJpbmc6OlNoZWxsUXVvdGU6OnNoZWxsX3F1b3RlICRkc2NfZm5hbWUpCiAgICAgICAgICAgICAgIC4gIiBcIlwkMVwiIgogICAgICAgICApLAorICAgICAgICAnLS1jdXN0b21pemUtaG9vaz1zbGVlcCAxMCcsCiAgICAgICAgICctLWN1c3RvbWl6ZS1ob29rPWNocm9vdCAiJDEiIHNoIC1jICInCiAgICAgICAgICAgLiAoCiAgICAgICAgICAgICBqb2luICcgJiYgJywK' | base64 -d | patch /usr/bin/debrebuild",
```

Comment:
I think the raw patch itself should be in code here and convert it to base64 inline. you could do that in the template itself or add a var somewhere to store the blob. also, please reduce the context lines from 3 before/after to maybe 1?